XSLT/tslaqdctomods.xsl

```diff
@@ -190,4 +192,19 @@
             <typeOfResource><xsl:value-of select="normalize-space(.)"/></typeOfResource>
         </xsl:for-each>
     </xsl:template>
+    
+    <!-- language -->
+    <xsl:template match="dc:language[1]">
+        <xsl:variable name="lang-tokens" select="tokenize(., ';')"/>
+        <xsl:for-each select="$lang-tokens">
+            <xsl:variable name="ltln" select="lower-case(normalize-space(.))"/>
+            <xsl:choose>
+                <xsl:when test="$ltln = $pLang/dltn:l">
+                    <languageTerm type="code" authority="iso639-2b">
```

Comment:
@markpbaggett - languageTerm needs to be nested within a language tag. Can you add this to the transform?

Reply:
Ugh sorry.  I was so busy focusing on the lack of namespaces that I didn't notice this was screwed up to begin with.  It should be good now.